src/Microsoft.VisualStudio.Editors/MyApplication/MyApplicationCodeGenerator.vb

```diff
@@ -196,24 +211,42 @@ Namespace Microsoft.VisualStudio.Editors.MyApplication
                 '  GENERATED CODE:
                 '    Me.IsSingleInstance = <True/False>
                 '    Me.EnableVisualStyles = <True/False>
-                '    Me.ShutDownStyle = ApplicationServices.ShutdownMode.xxx
-                '    Me.SaveMySettingsOnExit = <True/False>
+                '    Me.SaveMySettingsOnExit = <True/False>                
                 '
-                AddFieldPrimitiveAssignment(Constructor, "IsSingleInstance", MyApplication.SingleInstance)
-                AddFieldPrimitiveAssignment(Constructor, "EnableVisualStyles", MyApplication.EnableVisualStyles)
-                AddFieldPrimitiveAssignment(Constructor, "SaveMySettingsOnExit", MyApplication.SaveMySettingsOnExit)
+                AddFieldPrimitiveAssignment(Constructor, _singleInstanceFieldName, MyApplication.SingleInstance)
+                AddFieldPrimitiveAssignment(Constructor, _enableVisualStylesFieldName, MyApplication.EnableVisualStyles)
+                AddFieldPrimitiveAssignment(Constructor, _saveMySettingsOnExitFieldName, MyApplication.SaveMySettingsOnExit)
 
-                '
+                '    Me.ShutDownStyle = ApplicationServices.ShutdownMode.xxx
                 Dim EnumType As Type
                 EnumType = GetType(ApplicationServices.ShutdownMode)
                 If MyApplication.ShutdownMode = ApplicationServices.ShutdownMode.AfterAllFormsClose Then
-                    AddFieldAssignment(Constructor, "ShutDownStyle", EnumType, "AfterAllFormsClose")
+                    AddFieldAssignment(Constructor, _shutdownModeFieldName, EnumType, "AfterAllFormsClose")
                 ElseIf MyApplication.ShutdownMode = ApplicationServices.ShutdownMode.AfterMainFormCloses Then
-                    AddFieldAssignment(Constructor, "ShutDownStyle", EnumType, "AfterMainFormCloses")
+                    AddFieldAssignment(Constructor, _shutdownModeFieldName, EnumType, "AfterMainFormCloses")
                 Else
                     Debug.Fail("Unexpected MyApplication.ShutdownMode")
                 End If
 
+                '    Me.HighDpiMode = HighDpiMode.xxx
+                ' TODO: Add constants
+                Dim HighDpiValue As String
+                Select Case MyApplication.HighDpiMode
```

Comment:
Again, I don't know what's best practice here for the repo. 😄 
I would create a `Shared Dictionary(Of Integer, String)` with the values, and just index the respective strings out of it.
But it depends on what your team agrees on readability. In terms of performance, it certainly doesn't matter, since this is code which is only called once in a while. So, this is just an "in case suggestions".